### ISSUE TITLE
Moghouse Camera Issues

### ISSUE THREAD

**Issue by CdoubleOkay** [EdenServer/community#1524]
Ver: 30181205

Every time I enter Sandy moghouse on my Galka, I am Sandorian, my camera view glitches out. When I turn around it spins the other way. Just to exit the MH I have to snail crawl towards the door hoping my camera doesn't glitch another way. I have zoomed in as far as it will go to see if that helps navigating but the problem still occurs. Screenshots are not from First Person point of view.
![Ronswanson_2019 04 10_202331](https://user-images.githubusercontent.com/29466950/55922440-1890a000-5bcf-11e9-8b07-7f86dc048ee1.png)
![Ronswanson_2019 04 10_202337](https://user-images.githubusercontent.com/29466950/55922445-1e868100-5bcf-11e9-9621-cf8ed2868483.png)

### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x] I have provided my client version in the details. (type /ver into your game window)

**Comment by Far-The-Killer** [comment]
Need some details on this.

**Comment by CdoubleOkay** [comment]
Sorry didn't know I closed this... I will try and set up OBS to get a video for you. Other than that what details would you like?

**Comment by Setz0r** [comment]
i've reproduced this issue.  the camera is acting as if its colliding with furniture in the room when there is none.  this is probably an entity update bug letting the client think there's invisible stuff in the room when there is not.

**Comment by Quetzaxi** [comment]
I get this as well. Only when entering mog house from Port Sandy. No other mog house does it to me.

**Comment by godmode** [comment]
It may have something to do with that boat Pan saw in his mog house the other day. Awesome bug. Lol

**Comment by Setz0r** [comment]
boat npc doesn't have collision, furniture does.

**Comment by FallenXI** [comment]
There is a furniture camera collision setting somewhere in-game (misc, camera, etc) could be this?

**Comment by CdoubleOkay** [comment]
@FallenXI that setting helped my issue perfectly. However, I don't have any furniture in my mog so the underlying issue is still there

**Comment by CdoubleOkay** [comment]
Also, I am hearing the boat sfx as I enter my MH so the boat bug may still be occurring? Sounds like the boat is docking if that helps

**Comment by CdoubleOkay** [comment]
Not sure what the status is on this bug but after playing around I am able to find all edges of some type of furniture in my MH that shouldn't be there
![Ronswanson_2019 05 23_143433](https://user-images.githubusercontent.com/29466950/58291609-08521000-7d8c-11e9-9cc5-7efb19851ab0.png)
![Ronswanson_2019 05 23_143435](https://user-images.githubusercontent.com/29466950/58291610-08521000-7d8c-11e9-9db6-ceb18c4d0ed0.png)
![Ronswanson_2019 05 23_143436](https://user-images.githubusercontent.com/29466950/58291611-08521000-7d8c-11e9-8b16-385866e0233b.png)

**Comment by CdoubleOkay** [comment]
It seems the setting is no longer fixing my issue now as I get body blocked into my mog

**Comment by Setz0r** [comment]
what is your fps in mh

**Comment by CdoubleOkay** [comment]
Set to 60fps, holds steady at 58

**Comment by Setz0r** [comment]
if you switch to fps 30, does the invisible furniture collision still happen

**Comment by perkuprightmeow** [comment]
I play at 30FPS and I get this issue in the sandy moghouse every now & then.

**Comment by CdoubleOkay** [comment]
Yes it does. It seems every time I zone in and move relatively quickly the invisible furniture loads

**Comment by VeilEden** [comment]
Can confirm I only get this issue when the MH is “tied” to Port Sandy zone. It does not occur if I zoned into MH from any other Sandy zone. It acts as if there is an invisible line 1-2 yalms to the front of the Moogle, in the direction of the door.

**Comment by Althaia** [comment]
I have a Sandy alt and have noticed this with her. Like others said, it seems to only happen when my mog house is tied to Port San d'Oria.

I only notice it when I'm turning around and trying to run out of the mog house. Instead of getting to the door, I get blocked and the camera goes crazy/glitchy and won't let me go forward. I can still select the door by pressing F8 or tabbing over to the right target, so it's not really an issue, just an annoyance.
